### ISSUE TITLE
i18n/l10n/a11y Chapter

### ISSUE THREAD

**Issue by bebraw** [survivejs/webpack_react#14]
This is an advanced topic. It would be good to discuss various available libraries such as [react-a11y](https://www.npmjs.com/package/react-a11y) and basic approaches to i18n. Webpack's [i18n-webpack-plugin](https://github.com/webpack/i18n-webpack-plugin) should be discussed.

**Comment by vjancik** [comment]
For reference:
Also as an eslint plugin: https://www.npmjs.com/package/eslint-plugin-jsx-a11y

**Comment by bebraw** [comment]
Cool. Thanks for linking!
